css/style.css

```diff
@@ -253,14 +253,12 @@ a.footer-social:hover {
 .footer1 {
   position: sticky;
   bottom: 0;
-  padding: 25px 0;
+  padding: 30px 0;
   border: 1px solid #ff8008;
   background-color: #ff8008;
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
 }
```

Comment:
`<Footer>` is a valid HTML element. You can use that instead of creating a class called `footer` or in this case `footer1`. Also don't name things with numbers.